yasa/hypno.py

```diff
@@ -625,8 +643,8 @@ def plot_hypnogram(self, **kwargs):
         --------
         .. plot::
 
-            >>> from yasa import simulate_hypnogram
-            >>> ax = simulate_hypno(tib=480, seed=88).plot_hypnogram(highlight="REM"")
+            >>> from yasa import simulate_hypno
+            >>> ax = simulate_hypno(tib=480, seed=88).plot_hypnogram(highlight="REM")
```

Comment:
Nice catch!

btw -- I was going to ask anyways -- should this function be renamed as `simulate_hypnogram`? Just trying to think of consistency across functions (eg `plot_hypnogram`). Not a big deal but this is basically a brand new function so it seems like now would be the time to rename it if you want.

Reply:
Actually you're right, `simulate_hypnogram` would be better

Reply:
Done in latest commit!